### PR TITLE
WIP: py3

### PR DESCRIPTION
**Pull request by chiahaoliu**
Continuation of PR #5.

TODO:

- [x] Walk through the code and make sure it can be called inside python

- [ ] Docstring of package (at least the public-facing side) 

Attn: @CJ-Wright

### CODE REVIEW

File: tests/test_tools.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def test_estimateBaselineSlope(self):
         """
         slope = tools.estimateBaselineSlope(self.xobj, self.yobj)
         slopecalc = -4 * numpy.pi * self.rho0
-        self.assertAlmostEqual(slopecalc, slope, 2)
+        self.assertTrue(numpy.allclose(slopecalc, slope, 2))
```
[the remark this answers is not in the document]
Comment (chiahaoliu) [reply tests/test_tools.py]: Yeah, that's a good suggestion. Thanks.

This approach will not use ``assertTrue`` in ``unittest`` so in return, we lose some contexts in testing if it fails. (but it's helpful for moving to ``pytest``)

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply tests/test_tools.py]: Are we moving this project to pytest?

[the remark this answers is not in the document]
Comment (chiahaoliu) [reply tests/test_tools.py]: open question. In an ideal world, yes :p

**Comment by chiahaoliu** [comment]
@pavoljuhas 
It seems numpy doesn't allow rounding, even if it's scaler.

Here are the partial output from test

```
>       self.assertAlmostEqual(slopecalc, slope, 2)
E       TypeError: type numpy.ndarray doesn't define __round__ method
````

**Comment by pavoljuhas** [comment]
I have added a printout and `slope` is actually a 1-element array not a scalar.

`numpy.allclose` is better after all as it broadcasts them to compatible shape.  This form is however more like the original test:
```
self.assertTrue(numpy.allclose(slopecalc, slope, atol=0.01))
```

**Pull request closed without merging**